tools/tracer/README

```diff
@@ -0,0 +1,60 @@
+THIS DOCUMENT CONTAINS INFORMATION ON PRODUCTS IN THE DESIGN PHASE OF DEVELOPMENT.
+
+Copyright(c) 2014-2020 Intel Corporation. All Rights Reserved.
+
+
+Overview
+========
+
+Media SDK Tracer is a tool which permits to dump logging information from the calls
+of the application to the Media SDK library. Trace log obtained from this tool is a
+recommended information to provide to Media SDK team on submitting questions and
+issues.
+
+Media SDK tracer tool version is: v1.32.0. Note that 1.32 stands for the Media SDK API
+version this particular version of the Tracer is intended for.
+
+
+Prerequisites
+=============
+
+To use this version of the tracer, it is required to have the following products installed
+on the system:
+- Intel Media Stack: https://github.com/Intel-Media-SDK/MediaSDK/releases
+Please, refer to the release notes of the product on how to install it
+
+
+Installation
+============
+
+Generate config file for the tracer:
+$INSTALLDIR/tools/tracer/mfx-tracer-config --default
+$INSTALLDIR/tools/tracer/mfx-tracer-config core.type file
+$INSTALLDIR/tools/tracer/mfx-tracer-config core.log ~/mfxtracer.log
+$INSTALLDIR/tools/tracer/mfx-tracer-config core.lib /opt/intel/mediasdk/lib64/libmfxhw64-p.so.1.32
+
+Following this procedure you will generate a configuration file called '~/.mfxtracer' and set the tracer to dump
+logs to '~/mfxtracer_<PID>.log' files. You may adjust your configuration with 'mfx-tracer-config' tool.
+Run 'mfx-tracer-config -h' to get full list of supported options.
+
+
+Running
+=======
+
+For use the tracer, run the application with LD_PRELOAD:
+LD_PRELOAD=libmfx-tracer.so <app>
+
+Make sure that the tracer library is added to the search path:
+export LD_LIBRARY_PATH=$INSTALLDIR/tools/tracer:$LD_LIBRARY_PATH
+
+After each run of some Media SDK based application you should see traces in the configured log files.
+Note that the tracer library reads settings from ~/.mfxtracer located in the home directory of a current user.
```

Comment:
I think there was also some /etc/mfxtracer config. Was it? And that's what needs to be clarified.

Reply:
In the configure tool, we set the config path only like this:
`string config_path = HOME + string("/.mfxtracer");`